docs/code-quality/ca1801.md

```diff
@@ -35,6 +35,10 @@ This rule does not examine the following kinds of methods:
 
 - Methods used as event handlers.
 
+- Serialization constructors.
+
+- Serialization `GetObjectData` method.
```

Comment:
Should we add a hyperlink to https://docs.microsoft.com/dotnet/api/system.runtime.serialization.iserializable.getobjectdata?